testplan/common/remote/remote_service.py

```diff
@@ -60,6 +61,8 @@ class RemoteService(Resource, RemoteResource):
     :param rpyc_port: Specific port for rpyc connection on the remote host. Defaults to 0
         which start the rpyc server on a random port.
     :type rpyc_port: ``int``
+    :param sigint_timeout: number of seconds to wait between ``SIGINT`` and ``SIGKILL``
+    :type sigint_timeout: ``int``
```

Comment:
No need to add `:type ...:` directive when the type annotation is in place in the function signature. Can you please remove all of the `:type`-stuffy in the docstring here? I see all the parameters annotated.

Reply:
It is, but it is our old school doc approach. :type: duplicates content on API docs (that is already not the nicest thing to read) and does not support intellisense as seamless as type annotations.